grammars/clojure.cson

```diff
@@ -322,17 +322,6 @@
       {
         'include': '#sexp'
       }
-      {
-        'match': '(?<=\\()(.+?)(?=\\s|\\))'
```

Comment:
And why was this removed?

Reply:
This was placing the same `entity.name.function` to all function calls inside `defn`. Because the token is the same, there was no way to distinguish between function declaration and function call. I checked with other syntaxes and they seem to put `entity.name.*` on definitions only as well

Reply:
Hmm, I think this is worth keeping.  I checked language-javascript, language-java, and language-php and all use `entity.name.function` for function declarations and function calls but have an additional meta scope to differentiate between whether it's a declaration or call (usually `meta.function` vs `meta.{function|method}-call`).

Reply:
maybe you’re right. I was looking at Ruby. Returned it back